### PR TITLE
Resource hints

### PR DESCRIPTION
**Pull request by plehegar**
Addressed #119 

@igrigorik note that I removed support for a and area since those are not defined by resource hints.

### CODE REVIEW

File: sections/semantics-document-metadata.include [hunk 1]
```diff
@@ -680,14 +680,13 @@
     <a>supported tokens</a> are the keywords defined in
     <a>HTML link types</a> which are allowed on <code>link</code> elements, impact
     the processing model, and are supported by the user agent. The possible supported tokens are
-    <code>alternate</code>, <code>icon</code>, <code>prefetch</code>,
-    <code>stylesheet</code> and <code>next</code>.
+    <code>alternate</code>, <code>author</code>, <a>dns-prefetch</a>, <code>help</code>, <code>icon</code>, <code>license</code>, <code>next</code>, <a>preconnect</a>, <a>prefetch</a>, <a>prerender</a>, <code>prev</code>, <code>search</code>, and <code>stylesheet</code>.
```
Comment (travisleithead) [inline sections/semantics-document-metadata.include]: Please wrap lines at col 100 (for new code)

Comment (travisleithead) [inline sections/semantics-document-metadata.include]: This section includes the caveot that the list includes those tokens that **impact the processing model**. I don't believe that:
- author
- help
- license
- search
- prev/next (I think is was a bug that `next` was in the list before.)

File: single-page.bs [hunk 2]
```diff
@@ -374,6 +374,15 @@ urlPrefix: https://www.w3.org/TR/CSP3/#; type: dfn; spec: CSP
     text: frame-ancestors
     text: sandbox
 
+<!-- ********************************** RESOURCE-HINTS ********************************************* -->
+
+urlPrefix: https://www.w3.org/TR/resource-hints/#; spec: RESOURCE-HINTS;
+    type: dfn
```
Comment (travisleithead) [inline single-page.bs]: `type: dfn` can be on the line above (semi-colon separated). No reason to pull it down yet.

**Comment by travisleithead** [comment]
Reviewed! A few comments for @plehegar to follow-up on.

File: sections/semantics-document-metadata.include (hunk 1 above)
Comment (plehegar) [inline sections/semantics-document-metadata.include]: Good catch. I missed the distinction between rel's tokens and rel's supported tokens.

**Comment by travisleithead** [comment]
Beautiful. Thanks!

**Comment by igrigorik** [comment]
👍

**Pull request merged**